torch/csrc/profiler/util.cpp

```diff
@@ -312,13 +312,13 @@ std::string ivalueToStr(const c10::IValue& val, bool isString) {
     // json only takes "true" and "false" so we convert the string to lower case
     if (val.isBool()) {
       for (char& c : mystr) {
-        c = std::tolower(c);
+        c = static_cast<char>(std::tolower(c));
       }
     }
 
     // A double quote can cause issues with the chrome tracing so force
     // all inputs to not contain more than the 2 we add in this function
-    int count = std::count(mystr.begin(), mystr.end(), '\"');
+    auto count = std::count(mystr.begin(), mystr.end(), '\"');
```

Comment:
```suggestion
    auto count = std::count(mystr.begin(), mystr.end(), '"');
```
Use a char overload, right? No need for a string literal?